Dockerfile

```diff
@@ -10,6 +10,9 @@ RUN wget --no-check-certificate https://dl.xpdfreader.com/xpdf-tools-linux-4.03.
 
 RUN apt-get install libpoppler-cpp-dev pkg-config -y --fix-missing
 
+# Install Tesseract
+RUN apt-get install tesseract-ocr libtesseract-dev poppler-utils
```

Comment:
How large is the tesseract dependency? We need to be a bit careful to not blow up our public docker image.

Reply:
Following are the sizes of dependencies respectively:
tesseract-ocr => 15.8 MB
libtesseract-dev => 13.8 MB 
poppler-utils => 613 kB

are these sizes okay for the docker file or should we look for a workaround?

Reply:
That's ok :+1: